test/specs/scenarios/countrySelector.js

```diff
@@ -57,6 +57,18 @@ export const countrySelectorScenarios = async (lang) => {
         verifyInitialUIElements(countrySelectorCopy)
       })
 
+      it("should skip country selection screen with a preselected driver's license document type", async () => {
```

Comment:
Would be good to add a test in the `poa` scenarios to make sure the screen isn't showing for poa docs either